### ISSUE TITLE
Regression Tests – Using OpenFAST executable in Windows 7

### ISSUE THREAD

**Issue by bmazetto** [OpenFAST/openfast#528]
Hello Everyone,

I am trying to use the released binaries in a Windows 7, but none of the regression tests is running.

**Bug description**

All of the regression cases are failing. I’ve noticed that the simulations are not being executed and all the log files show the same message:

```
FAST_InitializeAll:FAST_Init:FAST_ReadPrimaryFile:Invalid numerical input for file
"..\build\reg_tests\glue-codes\openfast\5MW_Land_DLL_WTurb\5MW_Land_DLL_WTurb.fst" occurred while
trying to read NLinTimes.

FAST encountered an error during module initialization.
Simulation error level: FATAL ERROR

Aborting OpenFAST.
```

**To Reproduce**

I used a similar procedure as the one presented in the [link.](https://openfast.readthedocs.io/en/master/source/testing/regression_test.html#python-driver-with-a-given-executable. ) However, instead of using Visual Studio, I am trying to execute the commands through the Anaconda Prompt. My step-by-step procedure was:

1. I’ve created a folder in “C:\Program Files” called “openfast”;
2. Using the Anaconda Prompt, I’ve cloned the regression test package to the aforementioned folder (git clone --recursive https://github.com/openfast/openfast.git)
3. Next, I’ve used the commands:

- cd openfast
- git checkout dev
- git submodule update

4. After, I’ve installed the ifort redistribution (64 bits)
5. From the last openFAST release, I’ve copied “openfast_x64_double.exe” to the folder “C:\Program Files\Openfast\openfast\build\glue-codes\openfast” and the DLLs to the “C:\Program Files\Openfast\openfast\reg_tests\r-test\glue-codes\openfast\5MW_Baseline\ServoData” folder. 
6. Once everything was in place, I’ve tried to run the regression tests:

- cd “C:\Program Files\Openfast\openfast\reg_tests”
- python manualRegressionTest.py “C:\Program Files\Openfast\openfast\build\glue-codes\openfast\openfast_x64_double.exe” Windows Intel 1e-5”

**Expected behavior**
Getting results for each case and their comparison to the baseline.

**Screenshots, if applicable**
![image](https://user-images.githubusercontent.com/44245549/90283579-e1777200-de46-11ea-8178-c10593280f26.png)

**OpenFAST Version**
```
**************************************************************************************************
 OpenFAST

 Copyright (C)  National Renewable Energy Laboratory
 Copyright (C)  Envision Energy USA LTD

 This program is licensed under Apache License Version 2.0 and comes with ABSOLUTELY NO WARRANTY.
 See the "LICENSE" file distributed with this software for details.
 **************************************************************************************************

OpenFAST-v2.3.0
Compile Info:
 - Architecture: 64 bit
 - Precision: double
 - Date: Apr 03 2020
 - Time: 13:54:02
Execution Info:
 - Date: 08/14/2020
 - Time: 12:09:21-0300
```

**System Information (please complete the following information):**
 - OS: Windows 7 Enterprise - Service Pack 1

I’m using a corporate computer with Windows 7. Could you help me understand the error message? Is there any library missing?

**Comment by rafmudaf** [comment]
That error messaging is letting you know that the input files you've used are incompatible or otherwise in an incorrect format for the executable you've downloaded. Since you're using v2.3.0, you should be using the same version from the r-test repository: https://github.com/OpenFAST/r-test/tree/v2.3.0

Can you verify that you have the correct input file?

**Comment by andrew-platt** [comment]
As @rafmudaf mentioned, you are using the development version of the regression tests with the v2.3.0 version of the executable.

To run the regression tests for this version of OpenFAST, you will want to checkout version 2.3.0 in step 3 above:

```
git checkout v2.3.0
git submodule update
```

**Comment by bmazetto** [comment]
Thanks @rafmudaf and @andrew-platt! The difference between the r-test repository and the executable version was the reason for the problem I reported. As mentioned by Andrew, I was using the development version instead of the v2.3.0. After changing, I was able to execute the tests.

![Tests](https://user-images.githubusercontent.com/44245549/90388709-9e8fe700-e05e-11ea-9632-319d432b37ba.png)

There are some failures, but the comparison plots do not show relevant difference between the baseline and the local results.
